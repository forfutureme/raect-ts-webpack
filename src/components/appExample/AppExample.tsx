/**
 * @format
 * @Author: huweijian
 * @Date: 2020-04-17 15:28:14
 * @Desc: 通用组件例子
 */
import React from 'react'

interface AppExampleProps extends React.HTMLAttributes<HTMLElement> {}

const AppExample: React.FC<AppExampleProps> = ({children}) => {
  return <div className="app-example">{children}</div>
}

export default AppExample
