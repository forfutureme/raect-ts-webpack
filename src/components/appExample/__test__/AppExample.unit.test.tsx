/**
 * @format
 * @Author: huweijian
 * @Date: 2020-04-17 16:33:24
 * @Desc: AppExample单元测试
 */
import renderer from 'react-test-renderer'
import React from 'react'
import AppExample from '../AppExample'

describe('AppExample组件测试', () => {
  it('AppExample渲染测试', () => {
    const json = renderer
      .create(<AppExample>it is a AppExample</AppExample>)
      .toJSON()
    expect(json).toMatchSnapshot()
  })
})
