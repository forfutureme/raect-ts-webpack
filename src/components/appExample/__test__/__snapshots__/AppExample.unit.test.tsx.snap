// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AppExample组件测试 AppExample渲染测试 1`] = `
<div
  className="app-example"
>
  it is a AppExample
</div>
`;
